src/index.js

```diff
@@ -471,12 +474,131 @@ class XRState {
 }
 const xrState = GlobalContext.xrState = new XRState();
 
-if (nativeBindings.nativeVr) {
+if (nativeBindings.oculusVr) {
```

Comment:
If I'm reading this right that means when you compile with OVR, OpenVR would be disabled? Is that intended?